addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -290,6 +295,100 @@ database, without the s_newsletter_list class. See fixNewsletterListClass.
+                        <form id="newsletter_form" action="/website/form/" method="post" enctype="multipart/form-data" class="o_mark_required"
+                                data-mark="*" data-model_name="mailing.contact" data-success-mode="message" hide-change-model="true">
+                            <div class="s_website_form_rows row s_col_no_bgcolor">
+                                <div t-if="request.env['mailing.contact']._is_name_split_activated()" class="s_website_form_field s_website_form_model_required col-lg-6 mb-3" data-type="char" data-name="Field">
+                                    <label class="s_website_form_label" style="width: 200px" for="mailing_sub1">
+                                        <span class="s_website_form_label_content">First Name</span>
+                                        <span class="s_website_form_mark"> *</span>
+                                    </label>
+                                    <input id="mailing_sub1" type="text" class="form-control s_website_form_input"
+                                            name="first_name" placeholder="First Name" required="1"/>
+                                </div>
+                                <div t-if="request.env['mailing.contact']._is_name_split_activated()" class="s_website_form_field s_website_form_model_required col-lg-6 mb-3" data-type="char" data-name="Field">
+                                    <label class="s_website_form_label" style="width: 200px" for="mailing_sub2">
+                                        <span class="s_website_form_label_content">Last Name</span>
+                                        <span class="s_website_form_mark"> *</span>
+                                    </label>
+                                    <input type="text" class="form-control s_website_form_input"
+                                            name="last_name" placeholder="Last Name" required="1"/>
+                                </div>
+                                <div t-else="" class="s_website_form_field s_website_form_model_required col-lg-6 mb-3" data-type="char" data-name="Field">
+                                    <label class="s_website_form_label" style="width: 200px" for="mailing_sub1">
+                                        <span class="s_website_form_label_content">Name</span>
+                                        <span class="s_website_form_mark"> *</span>
+                                    </label>
+                                    <input id="mailing_sub1" type="text" class="form-control s_website_form_input" name="name" required="1" placeholder="John Smith"/>
+                                </div>
+                                <div t-attf-class="#{'col-lg-12' if request.env['mailing.contact']._is_name_split_activated() else 'col-lg-6'} mb-3 s_website_form_field s_website_form_model_required" data-type="email" data-name="Field">
```

Comment:
Are you sure this condition works ? 🤔 
The design doesn't change even if I change this option in the settings, am I missing something ? 😅 
<img width="1244" height="456" alt="Capture d’écran 2025-10-02 à 11 08 29" src="https://github.com/user-attachments/assets/b2ad77d3-d697-4c56-b2e0-cedd22624d16" />

Reply:
If you have already the snippet in the page, you'll need to add a new one to have the option active...

Not the best I agree but it's already like that on master 🤔